sym/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d81d1dd3-d5e8-4373-98a6-f95b797b3f9d",
+  "app_id": "sym",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Sym Audit Logs to Datadog",
+    "title": "Sym",
+    "media": [
+      {
+        "media_type": "video",
+        "caption": "A Sym Overview Video",
+        "image_url": "images/sym_video_thumbnail.png",
+        "vimeo_id": 123456789
```

Comment:
We have submitted our video to the Marketplace team, but are still waiting on a Vimeo ID.